onnxruntime/core/providers/dml/DmlExecutionProvider/src/GraphKernelHelper.cpp

```diff
@@ -105,11 +105,11 @@ namespace GraphKernelHelper
     bool GetGraphInputConstness(
         uint32_t index,
         const onnxruntime::OpKernelInfo& kernelInfo,
-        const onnxruntime::ConstPointerContainer<std::vector<onnxruntime::NodeArg*>>& fusedNodeInputDefs,
+        const gsl::span<const std::string>& fusedNodeInputArgOriginalNames,
```

Comment:
>& [](http://example.com/codeflow?start=42&length=1)

Spans are light (just 2 fields, a pointer and count) and meant to be passed directly, not by reference. Otherwise we're passing a pointer to a pointer.